components/card.scss

```diff
@@ -81,17 +94,20 @@
   padding: 0px 20px 12px;
 }
 
-.fullWithCardContentContainer {
+.fullWidthCardContentContainer {
   padding: 0px;
 }
 
 .cardGrid {
   display: grid;
-  grid-row-gap: 20px;
```

Comment:
Deleting these lines smushes the cards together in various places.

![Screenshot 2020-05-13 at 16 52 54](https://user-images.githubusercontent.com/5724925/81835506-3e0e7c80-953a-11ea-9e00-7078fa905c46.png)

Reply:
Ah I meant to revert that bit, adding in the second card type is a nicer alternative